src/codesy.coffee

```diff
@@ -40,7 +40,7 @@ codesy.isIssue = (url)->
   
 codesy.events.submit = (e)->
   e.preventDefault()
-  codesy.bid.update(codesy.form)
+  codesy.bid.update $ @
```

Comment:
Coffee script may be too terse here.  This is equivalent to $(this) which converts the calling form into the required jquery object.